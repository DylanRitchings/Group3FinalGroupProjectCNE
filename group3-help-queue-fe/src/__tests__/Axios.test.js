import React from 'react';
import renderer from 'react-test-renderer';
import createNewTicket from  '../queuedTickets';
import deleteTicket from  '../queuedTickets';
import updateTicketContents from  '../queuedTickets';
import updateTicketToCompleted from '../queuedTickets';




it('renders correctly', () => {
    const name = "Jest Test";
    const ticketDescription = "Jest Test description";
    const ticketTitle = "Jest Test Title";
    const complete = false;
    const tree = renderer.create(<createNewTicket name={name} ticketDescription={ticketDescription} ticketTitle={ticketTitle} complete={complete}/>).toJSON();
    expect(tree).toMatchSnapshot();
});

it('renders correctly', () => {
    const name = "Jest Test";
    const ticketDescription = "Jest Test description";
    const ticketTitle = "Jest Test Title";
    const tree = renderer.create(<updateTicketContents name={name} ticketDescription={ticketDescription} ticketTitle={ticketTitle}/>).toJSON();
    expect(tree).toMatchSnapshot(
    //     {
    //     name: "Jest Test",
    //     ticketDescription: "Jest Test description",
    //     ticketTitle: "Jest Test Title"
    // }
    );
});

it('renders correctly', () => {
    const id = 2;
    const tree = renderer.create(<deleteTicket id={id} />).toJSON();
    expect(tree).toMatchSnapshot();
});

it('updated to complete correctly', () => {
    const complete = true;
    const tree = renderer.create(<updateTicketToCompleted complete={complete} />).toJSON();
    expect(tree).toMatchSnapshot();
})