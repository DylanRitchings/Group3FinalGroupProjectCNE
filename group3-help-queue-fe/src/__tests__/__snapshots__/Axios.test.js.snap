// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<createNewTicket
  complete={false}
  name="Jest Test"
  ticketDescription="Jest Test description"
  ticketTitle="Jest Test Title"
/>
`;

exports[`renders correctly 2`] = `
<updateTicketContents
  name="Jest Test"
  ticketDescription="Jest Test description"
  ticketTitle="Jest Test Title"
/>
`;

exports[`renders correctly 3`] = `
<deleteTicket
  id={2}
/>
`;

exports[`updated to complete correctly 1`] = `
<updateTicketToCompleted
  complete={true}
/>
`;
